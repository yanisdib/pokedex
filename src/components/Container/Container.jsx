import styled from 'styled-components';


export default styled.div`
/* Landscape phones and down */
@media (min-width: 320px) {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    column-gap: 30px;
    align-items: center;
    justify-content: center;
    max-width: 1450px;
    min-height: 100vh;
    margin: auto;
}

/* Landscape phone to portrait tablet */
@media (min-width: 480px) {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    column-gap: 30px;
    align-items: center;
    justify-content: center;
    max-width: 1450px;
    min-height: 100vh;
    margin: auto;
}

/* Portrait tablet to landscape and desktop */
@media (min-width: 768px) {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    column-gap: 30px;
    align-items: center;
    justify-content: center;
    max-width: 1450px;
    min-height: 100vh;
    margin: auto;
}

/* Large desktop */
@media (min-width: 1200px) {
    display: flex;
    flex-direction: row;
    flex-wrap: wrap;
    column-gap: 30px;
    align-items: center;
    justify-content: flex-end;
    max-width: 1450px;
    min-height: 100vh;
    margin: auto;
}
`;