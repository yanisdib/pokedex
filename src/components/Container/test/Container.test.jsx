import { shallow } from 'enzyme';
import Container from '../Container';

describe('Container component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Container />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})