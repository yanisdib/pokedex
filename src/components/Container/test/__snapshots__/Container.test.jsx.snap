// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component renders correctly without crashing 1`] = `ShallowWrapper {}`;
